examples/pytorch/IterativePruning/README.md

```diff
@@ -0,0 +1,52 @@
+## Iterative Pruning
+Pruning is the process of compressing a neural network that involves removing weights from a trained model.
+It could be achieved by either removing the neurons in a specific layer or make the weights of the connections 
+that are nearer to zero as zero. The script is an example of later. Pruning a model has an impact on
+accuracy of the model, while it makes the model lightwieght. In this example, we train a Alexnet model to classify 
+CIFAR 10 dataset. Basemodel along with the parameters, metrics and summary are stored in mlflow.
+Subsequently, the trained Alexnet model stored in mlflow is downloaded and pruned iteratively by using the custom 
+inputs provided from the cli. Ax is a platform for optimizing any kind of experiment, including machine learning experiments,
+A/B tests, and simulations. [Ax](https://ax.dev/docs/why-ax.html) can optimize discrete configurations using multi-armed bandit optimization,
+and continuous (e.g., integer or floating point)-valued configurations using Bayesian optimization. The objective function of the experiment
+trails is "test_accuracy" based on which the model is evaluated at each trial and the best set of parameters are derived.
+AXClient is used to provide the initial pruning percentage as well as decides the number
+of trails to be run. The summary of the pruned model is captured in a seperate file and stored as an artifact in mflow.
+
+
+### Running the code to train the base model
+
+
+To run the training script directly with custom parameters:
+```
+python alexnet.py \
+    --max_epochs 5  \
+    --mlflow_experiment_name Prune \
```

Comment:
@ankan94 it seems mlflow_experiment_name has been removed from inputs of the script, please update the REAME/script, in case it should not be in the script please add the step to set the env variable for that as it been used in the script.

Reply:
Addressed.